### DISCUSSION TITLE
Multi-monolingual slides using Quarto?

### DISCUSSION THREAD

**Discussion by JuanAbasolo** [quarto-dev/quarto-cli#8985] [Q&A] [answered]
### Description

Multilingual slides using Quarto?

I have a presentation with interpretation into hegemonic languages. I would like my main screen slides to be in my minority and marginalized language, in which I will be giving the presentation, and provide audience members who speak majority languages with the URL where they can simultaneously view the translated slides.

It would seem possible that with revelajs + multiplex, my_slides+translation_1+translation_2 could be simultaneously broadcasted, each through its own URL.

If this is possible, how?

If it has not yet been implemented: I request that you evaluate the possibility of implementing multiple simultaneous outputs

**Comment by mcanouil** [comment, marked as answer]
I'm afraid it's not possible.
Maybe there is a Reveal.js plugin that exists already but I highly doubt that.

If you feel up to it, you can try to make one for your needs.

Note that Quarto/Pandoc only uses Reveal.js and does not really extend it.
It simply includes some plugins by default.

Note that Multiplex can only share the same content as the one "in front of you" not multiple versions of the same things especially because it would mean to stream to several different targets.

My two cents, you could use profile to create your slides in different languages and share the rendered slides after/before your talk.
But you can't multiplex to two targets as said earlier (at least in the current state of the multiplex plugin and as far as I know).


PS: I am switching this to Q&A as it is more than unlikely that Quarto team will develop a Reveal.js plugin.

**Reply by JuanAbasolo** [reply]
Thank you, @mcanouil , for the patient response.

Unfortunately, it's nowhere near my capabilities to technically solve the implementation.

I will investigate what door to knock on in revealjs, see if any open up.
